man/systemd.socket.xml

```diff
@@ -443,6 +443,14 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>MaxConnectionsPerSource=</varname></term>
+        <listitem><para>The maximum number of connections from one source to
+        simultaneously run services instances for. It's works in conjuction with
```

Comment:
What does "in conjuction with" mean in this context? Is it that it is ignored if MaxConnections is not set?

Also, it should be "It", not "It's"